test/actor_test.py

```diff
@@ -1290,6 +1296,19 @@ def inc(self):
     # Submit some new actor tasks.
     x_ids = [actor.inc.remote() for _ in range(5)]
 
+    # Make sure that getting the result raises an exception.
+    for _ in range(10):
+        for x_id in x_ids:
+            with pytest.raises(ray.worker.RayGetError):
+                # There is some small chance that ray.get will actually
+                # succeed (if the object is transferred before the raylet
+                # dies).
+                ray.get(x_id)
+
+    # Submit some more actor tasks after the actor has definitely been marked
+    # as dead.
+    x_ids = [actor.inc.remote() for _ in range(5)]
```

Comment:
This number used to be 100 I think. It got shortened to 5 because of the lengthening of the task lease I think.